src/main/java/com/icthh/xm/tmf/ms/product/web/rest/ProductDelegate.java

```diff
@@ -28,4 +29,13 @@ public ResponseEntity<List<Product>> listProduct(String fields,
                                                      Integer limit) {
         return ResponseEntity.status(HttpStatus.NOT_IMPLEMENTED).build();
     }
+
+    @Timed
+    @LogicExtensionPoint(value = "PatchProduct", resolver = OptionalProfileKeyResolver.class)
+    @PreAuthorize("hasPermission({'profile': @headerRequestExtractor.profile}, 'PRODUCT.UPDATE.PARTIAL')")
+    @PrivilegeDescription("Privilege to partial product")
+    @Override
+    public ResponseEntity<Product> patchProduct(String id, ProductUpdate productUpdate) {
```

Comment:
May be it will be great to have +1 test for this method.

Reply:
ok, what exactly this text should check?
@major13ua

Reply:
Done